### PR TITLE
CP Indigo: [package.xml] Update maintainers (Add a release-maintainer etc.) #649

### PR DESCRIPTION
**Pull request by davetcoleman**
Cherry-pick https://github.com/ros-planning/moveit/pull/649 to Indigo

Had to also cherry-pick https://github.com/ros-planning/moveit/commit/dc388399c0076c5e64e88a91151b0484218da749#diff-585bfefdbf6dfac0cf5509be970972e3 to avoid numerous merge conflicts

### CODE REVIEW

**Comment by v4hn** [comment]
@davetcoleman what's the CI failure about?

**Comment by davetcoleman** [comment]
I believe I have fixed it

File: moveit_planners/chomp/chomp_interface/package.xml [hunk 1]
```diff
@@ -5,7 +5,9 @@
   <version>0.7.12</version>
   <author email="gjones@willowgarage.com">Gil Jones</author>
 
-  <maintainer email="chitt@live.in">Chittaranjan S Srinivas</maintainer>
+  <maintainer email="chitt@live.in">Chittaranjan Srinivas Swaminathan</maintainer>
```
Comment (130s) [inline moveit_planners/chomp/chomp_interface/package.xml]: I was wondering what changed in this line. Turned out it's middle name.

File: moveit_ros/benchmarks/package.xml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
   <name>moveit_ros_benchmarks</name>
   <version>0.7.12</version>
-  <description>MoveIt tools for benchmarking</description>
 
-  <author email="isucan@google.com">Ioan Sucan</author>
-  <maintainer email="isucan@google.com">Ioan Sucan</maintainer>
-  <maintainer email="mferguson@fetchrobotics.com">Michael Ferguson</maintainer>
+  <description>Enhanced tools for benchmarks in MoveIt!</description>
 
-  <license>BSD</license>
```
Comment (v4hn) [inline moveit_ros/benchmarks/package.xml]: You accidentally removed the license tag here and CI complains about it...

Comment (130s) [reply moveit_ros/benchmarks/package.xml]: Thanks for the catch. Addressed. This is a cherrypick of a simpler change, so let me dismiss your review now that CI passed.

**Pull request merged**